src/strategies/index.js

```diff
@@ -17,6 +18,9 @@ module.exports = (env, rootUrl) => Object.keys(strategies)
     const strategy = strategies[type]
     const callbackURL = `${rootUrl}/${type}/callback`
     strategy.config = strategy.getConfig(env, callbackURL)
+    if (strategy.config && strategy.config.scope) {
+      strategy.config.scope = scopeHandler.decode(strategy.config.scope)
```

Comment:
This essentially will route any `LW_<provider>_SCOPE` through the same logic as a url parser so you can specify env variable scopes how you'd like